tasks/src/feature-extraction/about.md

```diff
@@ -0,0 +1,126 @@
+# Use cases
+
+## Sentence classification
+
+For sentence classification, we are interested in BERT’s hidden layer output (embeddings), so we get the sentence embedding using
+`feature-extraction` pipeline which contain contextual knowledge of English language that can be used as input for Tree based classifiers like Random Forest, XGBoost, CatBoost. 
```

Comment:
```suggestion
`feature-extraction` pipeline which contain contextual knowledge of English language that can be used as input for Tree based classifiers like Random Forest, XGBoost, CatBoost. You can check out [this tutorial](https://towardsdatascience.com/feature-extraction-with-bert-for-text-classification-533dde44dc2f) for more in depth explanation. 
```

Reply:
also I feel like it's any classifier really, so I'd rather avoid giving specifics and let people pick what they want.